pkgs/tools/graphics/lsix/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, makeWrapper, imagemagick }:
```

Comment:
```suggestion
{ lib, stdenvNoCC, fetchFromGitHub, makeWrapper, imagemagick }:
```

Reply:
I see [neofetch](https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/misc/neofetch/default.nix) uses `stdenvNoCC` but [any-nix-shell](https://github.com/NixOS/nixpkgs/blob/master/pkgs/shells/any-nix-shell/default.nix) uses `stdenv` so I was not sure whether one I should use. I thought it was whether there is a `Makefile` or not.